Tensile/Source/lib/include/Tensile/ContractionProblem.hpp

```diff
@@ -554,6 +569,8 @@ namespace Tensile
         size_t m_allocatedElementsNonBatchA;
         size_t m_allocatedElementsNonBatchB;
 
+        size_t m_workspaceSize = 0;
```

Comment:
This new member needs to be included in the hash and comparison functions in https://github.com/ROCmSoftwarePlatform/Tensile/blob/develop/Tensile/Source/lib/include/Tensile/ContractionProblem_Detail.hpp .

Reply:
hash and comparison function is updated.